mass_mailing_confirm/views/mass_mailing_view.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- © 2015 Antiun Ingenieria S.L. - Antonio Espinosa
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html). -->
+<openerp>
+<data>
+
+<record id="view_mail_mass_mailing_popup_form" model="ir.ui.view">
+    <field name="name">mass.mailing.form</field>
+    <field name="model">mail.mass_mailing</field>
+    <field name="priority">90</field>
+    <field name="inherit_id" ref="mass_mailing.view_mail_mass_mailing_form"/>
+    <field name="arch" type="xml">
+
+        <xpath expr="//header/button[4]" position="attributes">
```

Comment:
Sorry @hbrunn I don't know how to do better. This is the original inherited view:
```
        <record model="ir.ui.view" id="view_mail_mass_mailing_form">
            <field name="name">mail.mass_mailing.form</field>
            <field name="model">mail.mass_mailing</field>
            <field name="arch" type="xml">
                <form string="Mass Mailing">
                    <header>
                        <button name="action_test_mailing" type="object"
                            class="oe_highlight" string="Test Mailing"
                            attrs="{'invisible': ['|', ('state', '!=', 'draft'), ('body_html', '=', False)]}"/>
                        <button name="action_test_mailing" type="object"
                            attrs="{'invisible': ['&amp;', ('state', '=', 'draft'), ('body_html', '!=', False)]}"
                            string="Test Mailing"/>
                        <button name="send_mail" type="object"
                            attrs="{'invisible': ['|', ('state', '=', 'done'), ('body_html', '=', False)]}"
                            class="oe_highlight" string="Send to All"/>
                        <button name="send_mail" type="object"
                            attrs="{'invisible': ['&amp;', ('state', '!=', 'done'), ('body_html', '!=', False)]}"
                            string="Send to All"/>
                        <field name="state" widget="statusbar"/>
                    </header>
                    <div class="oe_form_box_info oe_text_center" attrs="{'invisible': [('scheduled', '=', 0)]}">
                        <p><strong>
                            <field name="scheduled" class="oe_inline"/>
                            emails are in queue and will be sent soon.
                        </strong></p>
                    </div>
                    <sheet>
(...)
```
As you can see there are two bottons, one with class ```oe_highlight``` and other without it. So I can't match by name, isn't it?